.github/workflows/capsule-cypress-manual-trigger.yml

```diff
@@ -97,17 +97,20 @@ jobs:
 
       - name: Install Vega binaries
         if: ${{ env.RUN_CAPSULE }}
-        run: go install -v ./cmd/vega
-        working-directory: vega
+        run: |
```

Comment:
follow up (non-blocking): mght be nice to have a shared base action rather than copying this across eventually.

Reply:
Yup, probably. There will be some little improvements later, I can add it then,